lib/AnalysisStructured/PtrAnalysis.cpp

```diff
@@ -16,6 +16,7 @@
 
 #include "llvm/ADT/TypeSwitch.h"
 #include "llvm/Support/Debug.h"
+#include <cassert>
```

Comment:
necessary?

Reply:
Yeah we use quite a few asserts here but never imported it